site/src/docs/components/navigation-link/tabs.mdx

```diff
@@ -0,0 +1,35 @@
+---
+slug: '/components/navigation-link/tabs'
+title: 'NavigationLink'
+---
+
+import { StatusLabel, Notification } from 'hds-react';
+
+import LeadParagraph from '../../../components/LeadParagraph';
+import Layout from '../../../components/layout';
+import PageTabs from '../../../components/PageTabs';
+
+# NavigationLink
+
+<StatusLabel type="alert">WIP</StatusLabel>
+<StatusLabel type="success" style={{ marginLeft: 'var(--spacing-xs)' }}>
+  Accessible
+</StatusLabel>
+
+<LeadParagraph>
+    Navigation links are designed to be used for site navigation inside navigation components.
+</LeadParagraph>
+
+<Notification label="A work in progress!" className="siteNotification">
+  The navigation link component is part of a collection of new navigation components that the HDS team is in the process of making. This means that this component is subject to change and we don't recommend using it in production just yet.
```

Comment:
Grammarly suggestion: "... component is part of a collection of new navigation components the HDS team is currently making. This means that this component is subject to change, and we don't recommend using it in production."